content/1_General/Cpp_Command.mdx

```diff
@@ -502,6 +502,8 @@ default.
 	</Resource>
 </Resources>
 
+Note that USACO does not have a stack size limit, aside from the usual 256 MB memory limit. See [this discussion of the stack size](https://forum.usaco.guide/t/floodfill-2016-january-silver-problem-3/1005/5) for more information.
```

Comment:
I don't think it's necessary to link the forum post here. All you need to take away from this section is that the code may crash locally but it will work if you submit it to USACO.

Reply:
^